pkg/cluster/internal/providers/podman/node.go

```diff
@@ -52,7 +52,7 @@ func (n *node) Role() (string, error) {
 func (n *node) IP() (ipv4 string, ipv6 string, err error) {
 	// retrieve the IP address of the node using podman inspect
 	cmd := exec.Command("podman", "inspect",
-		"-f", "{{range .NetworkSettings.Networks}}{{.IPAddress}},{{.GlobalIPv6Address}}{{end}}",
+		"-f", "{{.NetworkSettings.IPAddress}},{{.NetworkSettings.GlobalIPv6Address}}",
```

Comment:
so podmad does IPv6?
cool

Reply:
podman uses CNI so ...
this particular diff is just dealing with slightly different structs vs docker https://www.redhat.com/sysadmin/container-networking-podman

Reply:
CNI folks are our friends ... we'll have better support than libnetwork ... but I don't envision anything extraordinary different than this firewalld, IPv6 proxied portMapping, ... issues

Reply:
It won't have any IPv6 network in the default CNI config though.